src/Serialize.Linq.NetStandard16/ExpressionContext.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Serialize.Linq.Internals;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using Microsoft.Extensions.DependencyModel;
+
+namespace Serialize.Linq
+{
+    public class ExpressionContext : ExpressionContextBase
+    {
+        protected override IEnumerable<Assembly> GetAssemblies()
+        {
+#if !NETCOREAPP2_0 && !NETCOREAPP1_1 && !NETSTANDARD1_6
+            return AppDomain.CurrentDomain.GetAssemblies();
+#else
+            var depContext = DependencyContext.Default;
```

Comment:
`.deps.json` is a file in every netcore application. It lists the entire dependency closure for the application, and is merged with the currently used framework's `.deps.json` file in the installation directory.